app/javascript/components/top-corner-controller.jsx

```diff
@@ -47,4 +57,10 @@ TopCorner.propTypes = {
   showWeather: PropTypes.bool.isRequired,
   cityName: PropTypes.string.isRequired,
 };
-export default TopCorner;
+
+TopCorner.fragments = {
```

Comment:
How do fragments reduce the amount of times we are querying components in the side panel?